blivet/devices/storage.py

```diff
@@ -190,6 +190,18 @@ def raw_device(self):
         """ The device itself, or when encrypted, the backing device. """
         return self
 
+    @property
+    def sector_size(self):
+        """ Logical sector (block) size of this device """
+        if not self.exists:
+            return LINUX_SECTOR_SIZE
```

Comment:
Shouldn't this generally be derived from the parent devices' sector sizes like it's done for MD arrays?

Reply:
I've changed this to `parents[0].sector_size`. I think we can safely assume that for non-existing devices other than LVM and MD all parents must have same sector size so we can use only the first parent. (LVM now can't create such VG and MD is covered separately.) Thoughts?

Reply:
I think that's sufficient. The only other case coming to my mind are some non-LVM DM devices (think Docker, Stratis,...). But in those cases, it might very hard to tell what the "right" sector size would be.